queries.sql

```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth >= '2016-01-01' AND date_of_birth <= '2019-12-31';
+SELECT name FROM animals WHERE neutered = true AND escape_attemps < 3;
+SELECT date_of_birth FROM animals WHERE name NOT IN ('Agumon','Pikachu');
```

Comment:
- [ ] As mentioned by the previous review and for the project requirement to** list Date of birth of all animals named either "Agumon" or "Pikachu"**, I noticed that you didn't correct the change yet mentioned before. Till now, they are are still excluded and instead of `date_of_birth` you list the `weigth_kg` at line 11.

Reply:
@tresorsawasawa Hi! I have added in the file queries.sql in line 11, but kept my line 6 as original, because I was following the Project Requierements "List the date of birth of all animals named either "Agumon" or "Pikachu"; and didn't find the Querry requested. That's why I put both of them in the file. Please let me know if I should delete it. I put in the PRthe result of the query.

Reply:
Hi @HFG43 

At line 11, you listed `weigth_kg` instead of `date_of_birth`.

Reply:
@tresorsawasawa sorry! I confused the wording. I will change line 6. Thanks!